### PR TITLE
docker: release: get Docker image to build

### PR DESCRIPTION
**Pull request by akirillo**
This PR copies all of the requisite dependencies into the release Dockerfile such that the release image can properly build

### CODE REVIEW

File: docker/release/Dockerfile [hunk 1]
```diff
@@ -13,14 +13,30 @@ RUN cat rust-toolchain | xargs rustup toolchain install
...
+COPY ./gossip-api ./gossip-api
+COPY ./renegade-crypto ./renegade-crypto
+COPY ./srs ./srs
+COPY ./state ./state
+COPY ./statev2 ./statev2
+COPY ./system-bus ./system-bus
+COPY ./task-driver ./task-driver
+COPY ./test-helpers ./test-helpers
+COPY ./util ./util
+COPY ./workers ./workers
```
Comment (joeykraut) [inline docker/release/Dockerfile:39]: we can just copy in the whole repo now, no need to do one by one. this was done before these were individual crates

File: workers/network-manager/src/manager.rs [hunk 2]
```diff
@@ -267,7 +267,9 @@ impl NetworkManagerExecutor {
                             log::info!("Listening on {}/p2p/{}\n", address, self.local_peer_id);
                         },
                         // This catchall may be enabled for fine-grained libp2p introspection
-                        _ => {  }
+                        e => {
+                            log::info!("{:?}", e);
+                        }
```
Comment (joeykraut) [inline workers/network-manager/src/manager.rs:272]: seems this is likely not meant to be in this pr

**Review by joeykraut — APPROVED** [review]

File: workers/network-manager/src/manager.rs (hunk 2 above)
Comment (akirillo) [reply workers/network-manager/src/manager.rs:272]: ah ack

File: docker/release/Dockerfile (hunk 1 above)
Comment (akirillo) [reply docker/release/Dockerfile:39]: ack

**Pull request merged**